internal/anonssh/anonssh.go

```diff
@@ -48,6 +48,8 @@ type session struct {
 }
 
 func (s *session) request(ctx context.Context, req *ssh.Request) error {
+	_ = ctx // FIXME: not yet used
```

Comment:
This will probably be used later, so I think it's good that it's explicitly declared here that it's *currently* not used (this was flagged by lint)

Reply:
Works for me